public/js/actions/notifications.js

```diff
@@ -0,0 +1,32 @@
+import * as actionTypes from 'constants/action-types';
+
+
+export function addNotification(text, { type='info', userDismissable=false,
+                                 autoHide=true } = {}) {
+  return (dispatch) => {
```

Comment:
I was trying to work around the timeout issues. But yes this is no longer necessary. Good catch.